### PR TITLE
Harmonize proof computation with the rest of the code

### PR DESCRIPTION
**Pull request by cjpatton**
Stacked on #65 (merge that first).

What we previously referred to as `pi_prime` is now the "node proof". The node proof is corrected much like the seed, control bit, and payload.

Rename `pi_proof_adjustment()` to `adjusted_proof()`, getting rid of the last reference to "pi".

Finally, replace the zero seed in `eval_proof()` and `adjusted_proof()` with the proof we want to "hash forward". As of draft-irtf-cfrg-vdaf-12, XofTurboShake128 can handle variable length seeds.

### CODE REVIEW

File: poc/tests/test_vidpf.py [hunk 1]
```diff
@@ -209,36 +209,53 @@ def test_malformed_correction_word_payload(self):
...
+                # NOTE Even if true, this doesn't seem to imply an attack on
+                # robustness, at least not immediately. For privacy, it's
+                # crucial to make sure the client and honest aggregator agree
+                # on the public share: if the attacker can feed the aggregator
+                # a correction word that's been mangled in this way, then it
+                # can use it to distinguish whether a given prefix is on-path.
+                if vidpf.is_prefix(prefix, alpha, level):
+                    self.assertFalse(valid)
+                # else:
+                #     self.assertTrue(valid)
```
Comment (cjpatton) [inline poc/tests/test_vidpf.py:261]: @jimouris I haven't quite figured out where my analysis is going wrong.

Comment (cjpatton) [reply poc/tests/test_vidpf.py]: I figured out what's going on :)

Comment (jimouris) [reply poc/tests/test_vidpf.py]: Was the test wrong?

**Comment by cjpatton** [comment]
Rebased.

**Review by jimouris — APPROVED** [review]

**Pull request merged**